bootstrap.php

```diff
@@ -97,7 +97,7 @@ public function htmlUnclosed()
     $styles = $this->getStyles();
     $app_id = WordPressEscaper::escAttr($settings['app_id']);
     $secret = WordPressEscaper::escAttr($settings['secret']);
-    $secure_mode = WordPressEscaper::escAttr($settings['secure_mode']);
```

Comment:
We need a fallback on `$settings[:secure_mode]` if no value was found for `$settings[:identity_verification]`